crane-cloud/frontend
# chore: add new app revisions endpoint

# Description

- This PR adds the new endpoint that retrieves app revisions and this logic will conventionally be in redux
- For more than 10 revisions, the revisions listing should be paginated.

## Type of change


## Trello Ticket ID

https://trello.com/c/zAiIZXNf

## How Can This Been Tested?

- Run this branch locally and checkout the application settings page and also test out the scenario of having more than 10 revisions, to see the pagination.

## Checklist:
